### DISCUSSION TITLE
How to set short cut for Normcap installed from flatpak on Gnome?

### DISCUSSION THREAD

**Discussion by AlexNabokikh** [dynobo/normcap#451] [Q&A] [answered]
Hi All,

I know how to set custom shortcuts, but I cannot find the command that I should use for Normcap installed via Flatpak.
Can someone explain what I should do?

Thank you!

**Comment by dynobo** [comment, marked as answer]
Sure, you can use the following command to start it via flatpak:

```sh
flatpak run --command=normcap com.github.dynobo.normcap
```

Use that one for your custom shortcut. Hope that helps!

**Reply by AlexNabokikh** [reply]
That's exactly what I was looking for. Thank you so much!
